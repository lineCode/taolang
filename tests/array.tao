function main() {
    let a = [1,true,nil,"str",{},1+2*3,];
    println(a);
    println(a.length);
    println(a[3]);
    println([9][0]);
}
