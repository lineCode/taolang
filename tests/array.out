[1,true,nil,str,{},7]
6
str
9
